website/content/docs/nia/usage/run-ha.mdx

```diff
@@ -0,0 +1,184 @@
+* If failover occurs when no task is executing, CTS elects a new leader that attempts to run all tasks in once-mode.
+
+Note that driver behavior is consistent whether or not CTS is running in high availability mode.
+
+## Requirements
+
+* Verify that you have met the [basic requirements](/docs/nia/usage/requirements) for running CTS. 
+* CTS Enterprise 0.7 or later
+* Terraform CLI 0.13 or later
+* All instances in a cluster must be in the same datacenter.
+
+You must configure appropriate ACL permissions for your cluster. Refer to [ACL permissions](#) for details. 
+
+It’s not required, but we recommend specifying the [TFC driver](/docs/nia/network-drivers/terraform-cloud) in your CTS configuration if you want to run in high availability mode. 
+
+## Configuration
+
+Add the `high_availability` block in your CTS configuration and configure the required settings to enable high availability. Refer to the [Configuration reference](#) for details about the configuration fields for the `high_availability` block. 
+
+The following example configures high availability functionality for a cluster named `cts-cluster`: 
+
+<CodeBlockConfig filename="cts-config.hcl">
+
+```hcl
+high_availability {
+   cluster {
+      name    = "cts-cluster"
+      storage "consul" {
+        parent_path = "cts"
+        namespace = "ns"
+        session_ttl = "30s"
+      }
+   }
+
+   instance {
+      address = "cts-01.example.com"
+   }
+}
+```
+</CodeBlockConfig>
+
+### ACL permissions
+
+The `session` and `keys` resources in your Consul environment must have `write` permissions. Refer to the [ACL documentation](/docs/security/acl) for details on how to define ACL policies.
+
+If the `high_availability.cluster.storage.namespace` field is configured, then your ACL policy must also enable `write` permissions for the `namespace` resource.  
+
+## Start a new CTS cluster 
+
+We recommend deploying a cluster that includes three CTS instances. This is so that the cluster has one leader and two followers.
+
+1. Create an HCL configuration file that includes the settings you want to include, including the `high_availability` block. Refer to [Configuration Options for Consul-Terraform-Sync](/docs/nia/configuration) for all configuration options.
+1. Issue the startup command and pass the configuration file. Refer to the [`start` command reference](/docs/nia/cli/start#modes) for additional information about CTS startup modes. 
+  ```shell-session
+  $ consul-terraform-sync start -config-file ha-config.hcl
+  ```  
+1. You can call the `/status` API endpoint to verify the status of tasks CTS is configured to monitor. Refer to the [`/status` API reference documentation](/docs/nia/api/status) for information about usage and responses.
```

Comment:
Suggestion to also mention here that only the leader of the cluster will return a successful response.